TestApi/TestApi.Services/Services/BookingsApiService.cs

```diff
@@ -81,7 +81,7 @@ public async Task<List<Guid>> DeleteHearingsByPartialCaseTextAsync(DeleteTestHea
             }
             catch (BookingsApiException e)
             {
-                _logger.LogError($"Encountered error '{e.Message}'");
+                _logger.LogError("Encountered error '{message}'", e.Message);
```

Comment:
Exceptions should be logged like so
`_logger.LogError(e, "Encountered error '{message}'", e.Message);`

Reply:
Done!